src/content/developers/tutorials/run-light-node-geth/index.md

```diff
@@ -9,6 +9,66 @@ sidebar: true
 published: 2022-03-04
 ---
 
+<b> Please note that Geth light clients can be very slow to find peers. This is because they rely on full-node operators opting-in to be light servers that the light clients can request data from. There are usually only a small number of light servers available.
```

Comment:
```suggestion
**Please note that Geth light clients can be very slow to find peers. This is because they rely on full-node operators volunteering themselves as light servers from which the light clients can request data. There are usually only a small number of light servers available.**
```